.travis.yml

```diff
@@ -3,6 +3,9 @@ rvm:
  - 2.0.0
  - 2.1.2
  - jruby-9.0.0.0.pre1
+gemfile:
+- gemfiles/sidekiq310.gemfile
+- gemfiles/sidekiq334.gemfile
```

Comment:
Is it really wise to just collapse the version number like that?  What happens when there's version 3.10.0 and version 31.0.0?

The format Travis uses in their example seems to be `Gemfile.rails-2.3.x` which is both more legible in the build matrix and more robust, so to speak.